rules/Strict/Rector/AbstractFalsyScalarRuleFixerRector.php

```diff
@@ -5,13 +5,15 @@
 namespace Rector\Strict\Rector;
 
 use Rector\Core\Contract\Rector\AllowEmptyConfigurableRectorInterface;
-use Rector\Core\Rector\AbstractRector;
+use Rector\Core\Rector\AbstractScopeAwareRector;
 use Webmozart\Assert\Assert;
 
 /**
  * @see \Rector\Tests\Strict\Rector\BooleanNot\BooleanInBooleanNotRuleFixerRector\BooleanInBooleanNotRuleFixerRectorTest
+ *
+ * @internal 
```

Comment:
I would suggest to mark this class internal, so others do not subclass it.

its pretty specific and I think we should protect our users from using it